freedomofpress/securedrop-debian-packaging
# Added new public key for planned rotation and bumped securedrop-keyri…

# Status

Ready for review

# Description

Updates `securedrop-keyring`, adding the new public key for the  2021 rotation, and bumping its version to 0.1.5.

# Test plan
* clone https://github.com/freedomofpress/securedrop, verify and check out the tag `1.8.2`
* [ ] verify that the `securdrop` keyring (`../securedrop/install_files/securedrop/etc/apt/trusted.gpg.d/securedrop-keyring.gpg`) is identical to `securedrop-keyring/securedrop-keyring.gpg`
* [ ] build the `securedrop-keyring` package with `PKG_VERSION=0.1.5 make securedrop-keyring` successfully
* Set up a securedrop-workstation dev environment if not already available
* Copy the `securedrop-keyring` package to ` sd-small-buster-template` and sd-large-buster-template` and install it using `dpkg`, restart the templates and all workstation VMs
* [ ] Confirm that both the original and new public keys (fingerprints '22245C81E3BAEB4138B36061310F561200F4AD77' and '2359E6538C0613E652955E6C188EDD3B7B22E6A3') are available via `sudo apt-key list` on the template VMs and their child workstation VMs
* [ ] Confirm that `sudo apt-get update` completes without errors on the VMs
